packages/x-date-pickers/src/internals/hooks/useField/useField.utils.ts

```diff
@@ -820,3 +820,85 @@ export const parseSelectedSections = (
 
   return selectedSections;
 };
+
+export const getSectionValueText = <TDate extends PickerValidDate>(
+  section: FieldSection,
+  utils: MuiPickersAdapter<TDate>,
+  currentValue: TDate | null,
```

Comment:
```suggestion
  currentDate: TDate | null,
```

To keep the distinction between date and value (which can be a range)